react/features/invite/components/dial-in-info-page/DialInInfoApp.web.js

```diff
@@ -4,23 +4,20 @@ import { I18nextProvider } from 'react-i18next';
 
 import parseURLParams from '../../../base/config/parseURLParams';
 import { i18next } from '../../../base/i18n';
-
+import { isMobileBrowser } from '../../../base/environment';
```

Comment:
This is porbably where the bundle bloat comes from. Import directly from /functions

Reply:
changed as discussed.